tasks/Task.cpp

```diff
@@ -28,27 +28,271 @@ bool Task::configureHook()
 {
     if (! TaskBase::configureHook())
         return false;
+
+    // Initialize camera parameters.
+    // The library desconsider the tangencial values, that's why only the
+    // radial values are informed. 
+    frame_helper::CameraCalibration cal = _camera_parameters.get();
+    cam_cal = vpCameraParameters(cal.fx, cal.fy,cal.cx,cal.cy);
+
+    // Set the position of the square target in a frame which origin is
+    // centered in the middle of the square
+    double L = _marker_size.get()/2; // 2L is the square size
+    point[0].setWorldCoordinates(-L,  L, 0);
+    point[1].setWorldCoordinates( L,  L, 0);
+    point[2].setWorldCoordinates( L, -L, 0);
+    point[3].setWorldCoordinates(-L, -L, 0);
+
+    //Center of the target (reference point)
+    P.setWorldCoordinates(0, 0, 0);
+
+    // Define the task
+    task.setServo(vpServo::EYEINHAND_L_cVe_eJe) ;
+    task.setInteractionMatrixType(vpServo::CURRENT, vpServo::PSEUDO_INVERSE);
+
+    switch(_architecture.get().mode)
+    {
+        case IBVS:
+            task.addFeature(p,pd) ;
+            break;
+        case PBVS:
+            task.addFeature(t);
+            task.addFeature(tu);
+            break;
+        case HVS:
+            task.addFeature(p,pd) ;
+            task.addFeature(depth) ;
+            task.addFeature(tu) ;
+    }
+
+    // Set the Jacobian (expressed in the end-effector frame)
+    eJe.eye(6,6);
+    //eJe[3][3] = 0;
+    //eJe[5][3] = 1;
+    task.set_eJe(eJe);
+
+    // Set the pose transformation between the camera and the robot frame
+    Eigen::Affine3d camera2body;
+    if (_camera2body.get(base::Time::now(), camera2body, true))
```

Comment:
Don't read this in the configureHook, read it where you use it, and don't use interpolation (you actually cannot, since interpolation really demands that you use the stream aligner)